pkg/virt-api/api.go

```diff
@@ -732,6 +735,11 @@ func (app *virtAPIApp) Run() {
 	kubevirtCAConfigInformer := kubeInformerFactory.KubeVirtCAConfigMap()
 	kubeVirtInformer := kubeInformerFactory.KubeVirt()
 
+	// Wire up health check trigger
+	configMapInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
+		apiHealthVersion.Clear()
```

Comment:
I would also keep the default behavior of `WatchErrorhandler` here.

Reply:
Calling the default error handler now. Looks like it just does some logging at the moment but who knows what it will do in the future.